moveit_core/collision_detection_fcl/include/moveit/collision_detection_fcl/collision_common.h

```diff
@@ -190,10 +195,9 @@ struct FCLGeometry
   boost::shared_ptr<CollisionGeometryData>  collision_geometry_data_;
 };
 
-typedef boost::shared_ptr<FCLGeometry> FCLGeometryPtr;
-typedef boost::shared_ptr<const FCLGeometry> FCLGeometryConstPtr;
 typedef boost::shared_ptr<fcl::CollisionObject> FCLCollisionObjectPtr;
 typedef boost::shared_ptr<const fcl::CollisionObject> FCLCollisionObjectConstPtr;
+};
```

Comment:
I think this curly bracket is causing the build failures? Although I would expect more complaints from the compiler earlier.